app/addons/cors/__tests__/components.test.js

```diff
@@ -38,7 +38,7 @@ describe('CORS Components', function () {
       corsStore._configChanged = true;
```

Comment:
For the component tests you need to replace the `TestUtils.renderIntoDocument` with [enzyme](https://github.com/airbnb/enzyme). Here is a nice example https://github.com/apache/couchdb-fauxton/blob/master/app/addons/documents/__tests__/pagination-footer.test.js